src/scene/immediate.js

```diff
@@ -134,29 +134,29 @@ Object.assign(pc.Application.prototype, function () {
     }
 
     function _addLines(position, color, options) {
-        if (options.layer === undefined) options.layer = this.scene.layers.getLayerById(pc.LAYERID_IMMEDIATE);
-        if (options.depthTest === undefined) options.depthTest = true;
+        var layer = (options && options.layer) ? options.layer : this.scene.layers.getLayerById(pc.LAYERID_IMMEDIATE);
+        var depthTest = (options && options.depthTest !== undefined) ? options.depthTest : true;
+        var mask = (options && options.mask) ? options.mask : undefined;
```

Comment:
This one's a bit trickier. We don't document `pc.MeshInstance#mask` very well. There are some enums here:
https://github.com/playcanvas/engine/blob/master/src/scene/scene.js#L576
This line will convert 0 to undefined. Either way, the `if` statement below will fail. So I think this is fine.

Reply:
yep that's what I think as well, this should be fine.